app/scripts/store/dimStoreItems.directive.js

```diff
@@ -80,25 +79,49 @@
         '</div>'
       ].join('')
     };
-
-    function Link(scope, element, attrs) {
-      scope.vm.onDrop = function(id, $event, equip) {
-        var vm = scope.vm;
-
-        var srcElement = $('#' + id);
-
-        vm.moveDroppedItem(angular.element(srcElement[0])
-          .scope()
-          .item, equip);
-      };
-    }
   }
 
-  StoreItemsCtrl.$inject = ['$scope', 'loadingTracker', 'dimStoreService', 'dimItemService', '$q', '$timeout', 'toaster', 'dimSettingsService'];
 
-  function StoreItemsCtrl($scope, loadingTracker, dimStoreService, dimItemService, $q, $timeout, toaster, dimSettingsService) {
+  StoreItemsCtrl.$inject = ['$scope', 'loadingTracker', 'dimStoreService', 'dimItemService', '$q', '$timeout', 'toaster', 'dimSettingsService', 'ngDialog', '$rootScope'];
+
+  function StoreItemsCtrl($scope, loadingTracker, dimStoreService, dimItemService, $q, $timeout, toaster, dimSettingsService, ngDialog, $rootScope) {
     var vm = this;
 
+    // Detect when we're hovering a dragged item over a target
+    var dragTimer = null;
+    var hovering = false;
+    var dragHelp = document.getElementById('drag-help');
```

Comment:
instead of loading the element in angular, could we use an `ng-class` that looks something like `<div id='drag-help' ng-class='{"drag-dwell-activated": hovering}'></div>` or was there another reason that it needed to be done this way? minor thing, but was just curious for my own sake